source/expressions.tex

```diff
@@ -3988,8 +3974,10 @@
 \pnum
 For two similar types \tcode{T1} and \tcode{T2}\iref{conv.qual},
 a prvalue of type \tcode{T1} may be explicitly
-converted to the type \tcode{T2} using a \tcode{const_cast}. The result
-of a \tcode{const_cast} refers to the original entity.
+converted to the type \tcode{T2} using a \tcode{const_cast}
+if, considering the cv-decompositions of both types,
```

Comment:
Yes, it's already there just before that sentence. We strive to avoid multiple cross-references to the same target within the same paragraph.

Reply:
> avoid multiple cross-references to the same target ...

Agreed, but the targets are different in this case: "similar type" vs. "cv-decompositions".  I'd stick with what's in the paper - clearly the author thought a reference would be useful too.

Reply:
I think "similar type" and "cv-decomposition" are so close nowadays that you either know both, or none of the terms. I'll leave this for @zygoloid to decide.

Reply:
> "similar type" and "cv-decomposition" are so close nowadays

I respectfully disagree, but sure, let @zygoloid add the reference to make me happy :).  The edits are fine otherwise - this is the only remaining issue I have with this motion.

Reply:
In this case, the "cv-decompositions of both types" are produced as part of the process of determining whether the types are similar, for which we already gave a cross-reference, so I don't think repeating the cross-reference adds very much. "cv-decomposition" is in the index, so if a reader is still unclear on what it is, it's easy to look up.